pkgs/by-name/sp/spacetimedb/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  perl,
+  git,
+}:
+rustPlatform.buildRustPackage rec {
```

Comment:
You can use `finalAttrs` to make overriding easier

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

Reply:
buildRustPackage now support `finalAttrs`? :o